### PR TITLE
Xf86 input mtrack

### PR DESCRIPTION
**Pull request by dilyn-corner**
Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package

xf86-input-mtrack is a Xorg multitouch trackpad driver.

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/xf86-input-mtrack/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e 
+
+./configure \
+        --prefix=/usr
```
Comment (dylanaraps) [inline community/xf86-input-mtrack/build]: 4 spaces.

File: community/xf86-input-mtrack/build [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+make
+
```
Comment (dylanaraps) [inline community/xf86-input-mtrack/build]: Remove this newline.

File: community/xf86-input-mtrack/depends [hunk 3]
```diff
@@ -0,0 +1,2 @@
+xorg-server make
+xorgproto make
```
Comment (dylanaraps) [inline community/xf86-input-mtrack/depends]: Missing `mtdev`

File: community/xf86-input-mtrack/version [hunk 4]
```diff
@@ -0,0 +1 @@
+0.5.0 1
```
Comment (dylanaraps) [inline community/xf86-input-mtrack/version]: This should be `git`. My bad. @dilyn-corner

**Comment by dilyn-corner** [comment]
Yeah I noticed that the Git repo is 0.5.1 but the release is 0.5.0. Dunno why he did it like that.

**Pull request merged**